cron/cron.go

```diff
@@ -65,4 +65,5 @@ func (ct *Crontab) RemoveJob(id string) {
 	if job, ok := ct.jobs[id]; ok {
 		job.Deactivate()
 	}
+	logrus.Infof("Removed: %s", id)
```

Comment:
There is only debug logs in Deactivate function, there is no info logs that shows the job has been removed.